components/AvalancheForecastZoneMap.tsx

```diff
@@ -82,18 +84,20 @@ export const AvalancheForecastZoneMap: React.FunctionComponent<MapProps> = ({cen
       </SafeAreaView>
 
       {isLoading && (
-        <Center width="100%" height="100%" position="absolute" top="0">
+        <Center width="100%" height="100%" position="absolute" top={0}>
           <ActivityIndicator size={'large'} />
         </Center>
       )}
       {isError && (
-        <Center width="100%" position="absolute" bottom="6">
-          <Alert status={'warning'} px={6} py={4}>
-            <HStack space={2} flexShrink={1}>
-              <Alert.Icon mt="1" />
-              <Body>Unable to load forecast data</Body>
-            </HStack>
-          </Alert>
+        <Center width="100%" position="absolute" bottom={6}>
+          <VStack space={8}>
+            <Center bg={COLORS['warning.200']} px={24} py={16} borderRadius={4}>
+              <HStack space={8} flexShrink={1}>
+                <FontAwesome5 name="exclamation-triangle" size={16} color={COLORS['warning.700']} />
+                <Body>Unable to load forecast data</Body>
+              </HStack>
+            </Center>
+          </VStack>
```

Comment:
`Alert` on top, hand-crafted version below. Now the icon and text are actually centered correctly, and it's an actual warning icon. I'll refactor this once we need something similar elsewhere.

![image](https://user-images.githubusercontent.com/101196/212716653-ff60cef0-ee36-438a-b9fb-e6b17d96704b.png)